docs/Hardlinks/Check-if-hardlinks-are-working.md

```diff
@@ -8,21 +8,15 @@ _All methods require you to login to your terminal with PuTTY or similar softwar
 
 !!! warning
 
-    You can't create hard links for directories :bangbang:
-
-    You can't cross file systems :bangbang:
+    - You **CAN'T** create hard links for directories :bangbang:
+    - You **CAN'T**  hardlink across separate file systems, partitions, or mounts :bangbang:
+    - Some file systems, such as exFAT, are known not to support hard links and should be avoided (double-check if you are unsure!)
```

Comment:
are we going with `hardlinks` or `hard links`?

Reply:
i think everyone uses hardlinks, which one is correct, i wouldn't know

Reply:
the technically correct word is "hard link" / "hard links"

But as long as we *get consistent in this PR (we are not...)* then we can do a guide cleanup of hardlink/hardlinks => hard link / hard links later

Reply:
fixed with: https://github.com/TRaSH-Guides/Guides/pull/2004/commits/07ceefa9847776ba4e1f714900cf7b683ab367ee